### ISSUE TITLE
Since OC 8.0 the app doesn't show translation.

### ISSUE THREAD

**Issue by trockenasche** [raduvatav/polls#64]
I update OC from 7.0.4. to 8.0.0 and the poll app still works but everything is in english, doesn't show (in my case) german, what has made it before. Any suggestions?

**Comment by raduvatav** [comment]
I wasn't aware of a new version of OC, thanks for that info! :)  I'll look into it.

**Comment by trockenasche** [comment]
Any news?

**Comment by raduvatav** [comment]
@trockenasche nope :( I looked into it and couldn't fix it. It's described here:

https://doc.owncloud.org/server/8.0/developer_manual/app/l10n.html

After a lot of transi-thing and .po and .pot and whatever, I found this relevant:

> You then just need the .php, .json and .js files for a working localized app.

So I looked in other oc8 built-in apps and did the same, checked and double-checked and checked again. Files are OK, translation still not working. Actually, at least the .php files should have worked as in oc7 without changing anything, I don't see any change in syntax or logic - they just don't.

As far as I'm concerned, oc8 has the following "features" (compared to oc7):

- made it more difficult to find the installed / not installed apps (split on I don't know how many screens for no reason I can think of... "my fancy apps", "my important apps", "my not-so-recommended apps" or something...)
- broke translations (when searching the net for a fix I found lots of people being affected by the same issue)
- broke many existing apps (actually all I used besides the built-in ones - not only translations, apps not working any longer at all)

I'm sorry, I won't waste any more time on this.
